libdevcore/Exceptions.h

```diff
@@ -21,6 +21,10 @@
 
 #pragma once
 
+#ifdef __MIC__
+#pragma warning(disable:811) //exception specification for implicitly declared virtual function
```

Comment:
Again, please could you detail where the warning being masked is, and also add a push/pop pair?  Thanks!